CONTRIBUTING.md

```diff
@@ -124,10 +124,10 @@ Examples:
 
 #### Releases
 
-Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md). We recomment using [`bump2version`](https://github.com/c4urself/bump2version) instead of `tbump`, however. The example below updates the release to `v1.3.0` and outputs the results (`--verbose`):
+Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md). We recomment using [`bump2version`](https://github.com/c4urself/bump2version) instead of `tbump`, however. The example below updates the release to `1.3.0` and outputs the results (`--verbose`):
```

Comment:
```suggestion
Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md). We recommend using [`bump2version`](https://github.com/c4urself/bump2version) instead of `tbump`, however. The example below updates the release to `1.3.0` and outputs the results (`--verbose`):
```

Reply:
Don't we want to reference the local RELEASE.md file, at https://github.com/jupyterhub/ltiauthenticator/blob/main/RELEASE.md ?

Reply:
We do not! Thanks for the heads up.

Reply:
@consideRatio got my wires crossed there....all good now.